internal/backend/local/backend_plan_test.go

```diff
@@ -330,7 +330,7 @@ Terraform will perform the following actions:
         # (1 unchanged block hidden)
     }
 
-Plan: 1 to add, 0 to change, 1 to destroy.`
+Plan: 1 to add, 0 to import, 0 to change, 1 to destroy.`
```

Comment:
The format of this line is (unfortunately) subject to a compatibility constraint with Terraform Cloud, so we essentially can't change it.

Reply:
If this were to be conditional, so it would only display the import output if it had some import blocks in the config, would that work?

It would mean that it would only be versions of TFC that know how to render import blocks that would ever be processing this changed line. Plannable import will be launched with full support in TFC so we will be updating TFC with that kind of info anyway.

Reply:
I've made this condition inline with the discussion in the RFC.